.travis.yml

```diff
@@ -1,4 +1,3 @@
 sudo: false
-install: git clone --depth 1 https://github.com/sstephenson/bats.git
-script: PATH="./bats/bin:$PATH" test/run
```

Comment:
Ah, noticed that you changed the `test/run` file as well. I'll have to give this a try locally later when I have more time.